src/components/RegisterLink.test.tsx

```
@@ -0,0 +1,75 @@
+    cleanup(); // Clean up after each test
+  });
+
+  it.only("renders with children", () => {
+    const { container } = render(<RegisterLink>Register</RegisterLink>);
+
+    const button = screen.getByRole("button");
+    expect(button).toBeDefined();
+    expect(container.textContent).toBe("Register");
+  });
+
+  it.only("calls login when clicked", async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)